### ISSUE TITLE
Test and add support for high contrast mode.

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyone#429]
Test the theme with the OS high contrast mode.

Make sure that all controls and links work.

Resources:
https://adrianroselli.com/2017/11/os-high-contrast-versus-inverted-colors.html
https://www.a11yproject.com/posts/2020-01-23-operating-system-and-browser-accessibility-display-modes/

**Comment by carolinan** [comment]
I have tested Windows high contrast mode with Edge (chromium), Internet Explorer 11 and Firefox.
The visible focus indicator works well. The input fields like the search and comment form works well. 

High contrast black:
The controls works well with one exception:
In IE11, the plus and minus icon for the menu is not visible at all.
In Edge, and Firefox, the plus and minus icon is difficult to see because they are dark grey over black background.

High contrast white:
No issues found.

**Comment by ryelle** [comment]
I took a few screenshots, but TBH I'm not really sure what I'm looking for here. I noticed that Edge & Firefox use different link colors in High Contrast Black (which makes FF almost unreadable, but it was better on my PC monitor than it looks in the screenshot). I don't know if the +/- icon is an issue since it seems fairly high contrast against the black, but I didn't check the contrast ratio.

| Firefox | Edge |
|--------|------|
| ![high-contrast-black-menu](https://user-images.githubusercontent.com/541093/95924504-b2697800-0d85-11eb-899a-1d2cf9ec1b3a.jpg) | ![high-contrast-black-menu-edge](https://user-images.githubusercontent.com/541093/95924503-b2697800-0d85-11eb-94ce-19ec48aba3d7.jpg) |

In High Contrast White, I noticed the footer links have this double underline. I didn't notice it anywhere else, but didn't look very hard.

![high-contrast-white-double-link](https://user-images.githubusercontent.com/541093/95924505-b2697800-0d85-11eb-8b6b-ddfc35191532.jpeg)

There's an extra border under the entry meta…

![high-contrast-white-extra-border](https://user-images.githubusercontent.com/541093/95924506-b3020e80-0d85-11eb-880b-19255684ccda.jpg)

The menu looks fine in White 👍 

![high-contrast-white-menu](https://user-images.githubusercontent.com/541093/95924507-b3020e80-0d85-11eb-984a-491ee893b705.jpg)

**Comment by carolinan** [comment]
I tested the four default color schemes again because I did not get the same result as in the screenshots above in my first test. 

The plus and minus icon **is** visible in IE11.
The plus minus icon color passes level AA.

The double underline on some links is because those links have both an underline and a transparent border. There should already be an issue to remove these unused borders.

My suggestions is that we close this without making any changes.
Firefox has browser settings where the user can customize the colors further.

**Comment by aristath** [comment]
> My suggestions is that we close this without making any changes.
> Firefox has browser settings where the user can customize the colors further.

+1

**Comment by afercia** [comment]
I think the double underline noticed by @ryelle comes from this ruleset:

```
a.custom-logo-link, nav a {
    border-bottom: 1px solid transparent;
    text-decoration: none;
}
```

where the transparent border becomes visible in High Contrast Mode (this is expected). Seems to me this transparent border is meant to be used for the theme's navigation menus only. However, since some widgets are now wrapped in a nav element because of `navigation-widgets` support, it's also applied to the links within these widgets. Maybe resetting the border for the widgets would be nice and not so hard to implement.

**Comment by carolinan** [comment]
Closing, the double underline/border combination has been removed
